shared/pingpong/config.go

```diff
@@ -94,6 +95,7 @@ var DefaultConfig = PpConfig{
 	RandomizeFee:    false,
 	RandomizeAmt:    false,
 	RandomizeDst:    false,
+	MaxRandomDst:    200000,
```

Comment:
this change default value might change nightly results. @egieseke @algobarb please be prepared